### PR TITLE
metadata for examples/models/file/calendars.py added

### PR DESCRIPTION
**Pull request by pranshu-kumar**
- [x] issues: addresses #11765

### CODE REVIEW

**Comment by bryevdv** [comment]
@pranshu-kumar Thanks for the PR this looks good, there's just a couple of small linter issues to sort out:

* some lines have trailing whitespace. Our lint rules reject this. Just need to delete any trailing whitespace (your editor may have a setting to do this automatically)
* the APIs line is longer than our usual max line length. You can add a comment ` # noqa: E501` to the end of the line and that will allow an excpetion

File: examples/models/file/calendars.py [hunk 1]
```diff
@@ -1,3 +1,13 @@
+'''A rendering of the 2014 monthly calender. 
+This example demonstrates the usage of plotting several plots together using ``gridplot``.
```
Comment (bryevdv) [inline examples/models/file/calendars.py:2]: small suggestion:

```suggestion
'''A rendering of the 2014 monthly calendar. 

This example demonstrates the usage of plotting several plots together using ``gridplot``.
```

**Comment by pranshu-kumar** [comment]
@bryevdv Thanks for the suggestions. I have made the changes.

File: examples/models/file/calendars.py [hunk 2]
```diff
@@ -1,3 +1,16 @@
+''' A rendering of the 2014 monthly calender.
```
Comment (bryevdv) [inline examples/models/file/calendars.py:1]: typo:
```suggestion
''' A rendering of the 2014 monthly calendar.
```

File: examples/models/file/calendars.py [hunk 3]
```diff
@@ -1,3 +1,16 @@
...
+plots together using ``gridplot``.
+
+A hover tooltip displays the US holidays on the significant dates.
+
+.. bokeh-example-metadata::
+    :sampledata: us_holidays
+    :apis: bokeh.layouts.gridplot, bokeh.models.tools.HoverTool, bokeh.models.plots.Plot, bokeh.models.glyphs.Rect, bokeh.models.glyphs.Text, bokeh.document.document.Document # noqa: E501
+    :refs: :ref:`userguide_layout` > :ref:`userguide_layout_gridplot`, :ref:`userguide_tools` > :ref:`userguide_tools_hover_tool`
+    :keywords: gridplot, hover, tooltip
+'''
```
Comment (bryevdv) [inline examples/models/file/calendars.py:13]: We typically leave a blank line at the bottom of every docstring, though I am not sure this documented anywhere (cc @tcmetzger)
```suggestion
    :keywords: gridplot, hover, tooltip

'''
```

**Comment by bryevdv** [comment]
Thanks @pranshu-kumar I noticed two more tiny suggestions if you have time to apply them (or I can commit them from the PR)

**Comment by pranshu-kumar** [comment]
I committed the changes from the PR. Thanks for the suggestions

**Review by bryevdv — APPROVED** [review]

**Comment by bryevdv** [comment]
Thanks for the PR @pranshu-kumar !

**Pull request merged**